File: src/se/axelbengtsson/kinapokerscorecalculator/BonusType.java
package se.axelbengtsson.kinapokerscorecalculator;
/**
 * Describe the Bonus and the point that is added per player.
 */
public enum BonusType {
  NoBonus(0),
  Kind(3),
  FullHouse(2),
  FourOfAKind(4),
  StraightFlush(5),
  ScoopUp(3),
  NotPlay(-3),
  Automatic(3),
  Dragon(13),
  ColorDragon(39);

  final int points;
  BonusType(int points) {
    this.points = points;
  }
}

File: app/src/main/java/se/axel/bengtsson/kinapokerscorecalculator/BonusType.kt
package se.axel.bengtsson.kinapokerscorecalculator

/**
 * Describe the Bonus and the point that is added per player.
 */
enum class BonusType(val points: Int, val short:String) {
  NoBonus(0, "-"),
  Kind(3, "3K"),
  FullHouse(2, "FH"),
  FourOfAKind(4, "4K"),
  StraightFlush(5, "SF"),
  ScoopUp(3, "S"),
  Win(1, "W"),
  NotPlay(-3, "NP"),
  Automatic(3, "A"),
  Dragon(13, "D"),
  ColorDragon(39, "CD");
}
